linelist_investigations/report_sources/aaa_clean_linelist_2019-10-18.Rmd

```diff
@@ -373,8 +373,14 @@ This section defines age classes - see code for details.
 ```{r clean_age}
 
 x <- x %>%
-  mutate(
-      age = as.numeric(gsub("_", ".", age)),
+  mutate(age = gsub("_", ".", age))
+
+# line below converts values that were initially in scientific notation (indicative of age < 1 year) to # prevent NA's from arising when calling "age = as.numeric(age)"
```

Comment:
use 2 ## for comments raher than #, as a single one is indented differently on different editors